hbase-client/src/main/java/org/apache/hadoop/hbase/BalanceRequest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase;
```

Comment:
Moved to `o.a.h.h.client`. Sorry, not sure how I accidentally put it in the hbase package.

There doesn't appear to be any `client.admin` package. I could put this class in a new `client.admin` package, but it would be the only thing there. Otherwise there are many other admin-related classes in `o.a.h.h.client`, including Admin itself. So might be weird to do this.